aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -104,27 +104,95 @@ To define a JS initializer, add a JS module to the project named `{NAME}.lib.mod
 
 The module exports either or both of the following conventional functions:
 
+:::moniker-end
+
+:::moniker range=">= aspnetcore-8.0"
+
+<!-- UPDATE 8.0 Remove NOTE -->
+
+> [!NOTE]
+> JS initializers for Blazor Web Apps is an upcoming feature that will appear with the release of .NET 8 in mid-November.
+
+For Blazor Web Apps:
+
+* `beforeWebStart(options)`: Called before the Blazor Web App starts. For example, `beforeStart` is used to customize the loading process, logging level, and other options. Receives the Blazor options (`options`).
+* `afterWebStarted(blazor)`: Called after the Blazor Web App is ready to receive calls from JS. For example, `afterWebStarted` is used to initialize libraries by making JS interop calls and registering custom elements. The Blazor instance is passed to `afterWebStarted` as an argument (`blazor`).
```

Comment:
JS interop can't yet be performed at this stage. That's only possible after an interactive Server or WebAssembly runtime starts up.

Reply:
What should the description be?

Reply:
Maybe something like, "Called after all `beforeWebStart` promises resolve. For example, `afterWebStarted` can be used to register Blazor event listeners and custom event types".